privacyidea/lib/eventhandler/base.py

```diff
@@ -140,6 +141,11 @@ def conditions(cls):
                           "not exist in the userstore anymore."),
                 "value": ("True", "False")
             },
+            CONDITION.TOKEN_IS_EXPIRED: {
+                "type": "str",
+                "desc": _("The token has an expired validity period"),
```

Comment:
I think it could also be, that the token is not valid, yet. So "expired" might be the wrong word here.
Do you think this may lead to confusions?

Because this condition could also be used, if the validity period of the token starts next week...